setup.py

```diff
@@ -34,6 +34,7 @@
         'django-timezone-field>=2.0rc1',
         'django-localflavor>=1.1',
         'swapper>=0.3.0',
+        'django-recaptcha',
```

Comment:
Yes, you are right. However, `axes` library we use is not the one on PyPi, we are currently using a forked customized version of this library to resolve an issue with IP tracking. We need to talk to dana about how to resolve such an issue. I am thinking about creating our own version of this library and publish it to PyPi as it is licensed under MIT